adal/authentication_context.py

```diff
@@ -67,8 +70,20 @@ def __init__(self, authority, validate_authority=None, cache=None):
         self._oauth2client = None
         self.correlation_id = None
         env_value = os.environ.get('ADAL_PYTHON_SSL_NO_VERIFY')
+        if api_version is not None:
+            warnings.warn(
+                """The default behavior of including api-version=1.0 on the wire
+                is now deprecated.
+                Future version of ADAL will change the default value to None.

```

Comment:
I don't think that any developers are going to understand that warning.  Client developers don't care about the protocol at all, and also don't have visibility in to the claims in the token.  They may know nothing about the effect of the api-version QP on the claims in the token and how that might break a service.  I don't think that I would put that warning in.